store/sdk/src/main/java/org/apache/carbondata/sdk/file/CarbonWriterBuilder.java

```diff
@@ -445,6 +449,21 @@ public CarbonWriterBuilder withBlockletSize(int blockletSize) {
     return this;
   }
 
+  /**
+   * To set the blocklet size of CarbonData file
+   *
+   * @param pageSizeInMb is page size in MB
+   *                     default value is 1 MB.
+   * @return updated CarbonWriterBuilder
+   */
+  public CarbonWriterBuilder withPageSizeInMb(int pageSizeInMb) {
+    if (pageSizeInMb < 1 || pageSizeInMb > 1755) {
```

Comment:
better to use variable "TABLE_PAGE_SIZE_MAX_INMB "

Reply:
this code is from PR2814, this PR dependency on it. it will be removed after PR2814 is merged